templates/tab/js/default/src/components/sample/PersonCardGraphToolkit.jsx

```diff
@@ -0,0 +1,28 @@
+import { PersonCard } from "@microsoft/mgt-react";
+import { useTeamsFx } from "./lib/useTeamsFx";
+
+export function PersonCardGraphToolkit(props) {
+  const { themeString } = useTeamsFx();
+
+  return (
+    <div className="section-margin">
+      <p>This example uses Graph Toolkit's
+        <a href="https://docs.microsoft.com/en-us/graph/toolkit/components/person-card" target="_blank" rel="noreferrer">person card component</a>with
+        <a href="https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider" target="_blank" rel="noreferrer">TeamsFx provider</a>to show person card.
```

Comment:
This example uses Graph Toolkit's[person card component](https://docs.microsoft.com/en-us/graph/toolkit/components/person-card)with[TeamsFx provider](https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider)to show person card.

Add space between text and link.

Reply:
Updated